experimental/dds/tree2/src/test/testChange.spec.ts

```diff
@@ -4,23 +4,21 @@
  */
 
 import { strict as assert } from "assert";
-import { FieldKinds, NodeChangeset } from "../feature-libraries";
-import { makeAnonChange, TreeSchemaIdentifier, Delta } from "../core";
+import { FieldKinds, singleTextCursor } from "../feature-libraries";
+import { makeAnonChange, TreeSchemaIdentifier, Delta, FieldKey } from "../core";
 import { brand } from "../util";
 import { TestChange } from "./testChange";
 
 const nodeType: TreeSchemaIdentifier = brand("Node");
 const fieldHandler = FieldKinds.value.changeHandler;
 const tree1 = { type: nodeType, value: "value1" };
 const tree2 = { type: nodeType, value: "value2" };
-const nodeChange1: NodeChangeset = { valueChange: { value: "value3" } };
-const nodeChange2: NodeChangeset = { valueChange: { value: "value4" } };
-const nodeChange3: NodeChangeset = { valueChange: { value: "value5" } };
 
-const change1WithChildChange = { value: tree1, changes: nodeChange1 };
-const childChange1 = { changes: nodeChange1 };
-const childChange2 = { changes: nodeChange2 };
-const childChange3 = { changes: nodeChange3 };
+const change1WithChildChange = { value: tree1, changes: TestChange.mint([0, 1], 2) };
```

Comment:
Looks like there are some unused variables in here that could removed instead of updated.